WalletWasabi/WabiSabi/Backend/Rounds/RoundParameters.cs

```diff
@@ -13,7 +14,8 @@ public class RoundParameters
 			Network network,
 			WasabiRandom random,
 			FeeRate feeRate,
-			Round? blameOf = null)
+			Round? blameOf = null,
+			Prison? prison = null)
```

Comment:
This doesn't look good to me. The `prision` is received only to be used in a predicate to fill the `BlameWhitelist` list that is used as a synonym in the `Round.BlameWhitelist` that is only used in by `Arena`. 

If we need the `Prision` instance only for this then we should make the `BlameWhitelist` a method `GetBlameWhitelist(Prision?)`.

Reply:
that would be a big change (tests etc), I already have a WIP cleanup that I want to merge after this but which was a headache to include in #6065, so i'll handle it as soon as this and #6162 are merged

Reply:
I am not sure about that, it is used in very few places.

Reply:
https://github.com/zkSNACKs/WalletWasabi/pull/6172